src/watchman_watcher.js

```diff
@@ -271,6 +281,7 @@ WatchmanWatcher.prototype.close = function(callback) {
 
 function handleError(self, error) {
   if (error != null) {
+    console.log('[sane] Watchman: ' + error.message);
```

Comment:
Good point. Since this is bubbled up through the error event it'll either be caught by the client or throw. This seems redundant. I'll remove it.